redfishMockupServer.py

```diff
@@ -701,6 +730,10 @@ def do_POST(self):
                 elif "TelemetryService/Actions/TelemetryService.SubmitTestMetricReport" in self.path:
                     r_code = self.handle_telemetry(data_received)
                     self.send_response(r_code)
+                # UpdateService (SimpleUpdate is handled part of the Actions, so only multipart is here)
+                elif "UpdateService/update-multipart" in self.path:
+                    logger.info("TBD: handle multipart")
```

Comment:
Is this meant to start a task based on the issue filed for this support?

Reply:
Yes, this should start a new task
Should be in separate pull request